src/main/scala/pl/edu/agh/iet/akka_tracing/TracedActor.scala

```diff
@@ -6,25 +6,8 @@ import akka.actor.Actor
 
 case class MessageWrapper(id: UUID, msg: AnyRef)
 
-//trait DistributedStackTraceMessage {
-//  val stackTrace = Thread.currentThread().getStackTrace
-//}
-
-trait TracedActor {
-  actor: Actor =>
-
+trait TracedActor extends Actor {
```

Comment:
This is actually a good question. The answer is: technical reason. I needed to have `aroundReceive` method here to be able to catch the call in aspect (even if the only thing it's calling is super). This method is private or package private so I could not have `actor.aroundReceive` here. So I needed to extend this trait to do this.